src/renderer/src/routes/app/projects/$projectId/-displayed-data/list.tsx

```diff
@@ -426,7 +426,7 @@ function ObservationCategory({
 						projectId={projectId}
 						iconDocumentId={categoryIconDocumentId}
 						altText={t(m.categoryIconAlt, { name })}
-						imageStyle={{ aspectRatio: 1, maxHeight: 12, objectFit: 'cover' }}
```

Comment:
fixes a very subtle bug where rendering the svg icon in the observation list row would render 0x0px when an attachment preview is present.